releaseArtMaker.sh

```diff
@@ -0,0 +1,126 @@
+current_github_release_version=$(grep 'libVersion =' scripts/publish.gradle.kts | sed 's/[^0-9.]//g')
+
+echo "Current Release Version is: $current_github_release_version"
+
+# Get the Release Type as User Input...
+echo "Please select the type of Release:"
+echo "1) Major (Incompatible API Changes)"
+echo "2) Minor (Backward Compatible Functionality)"
+echo "3) Patch (Backward Compatible Bug Fixes)"
+read -rp "Please choose your Release (1/2/3): " github_release_type_choice
+
+case $github_release_type_choice in
+    1)
+        github_release_type="Major"
+        ;;
+    2)
+        github_release_type="Minor"
+        ;;
+    3)
+        github_release_type="Patch"
+        ;;
+    *)
+        echo "Invalid Choice"
+        exit 1
+        ;;
+esac
+
+# Calculate the next GitHub Release version...
+next_github_release_version=$(get_next_release_version "$current_github_release_version" "$github_release_type")
+echo "The next GitHub Release version will be: $next_github_release_version"
+
+# Get the GitHub Release Notes...
+echo "Enter the GitHub Release Notes (end the input using an empty line by pressing ENTER twice):"
+github_release_notes=""
+while IFS= read -r line; do
+    [[ -z "$line" ]] && break
+    github_release_notes+="$line"$'\n'
+done
+
+# Escape the double quotes in the Release Notes for JSON Compatibility...
+refined_github_release_notes=$(echo "$github_release_notes" | sed 's/"/\\"/g')
+
+# Create the Changelog Link...
+previous_github_release_version="v$current_github_release_version"
+new_github_release_version="v$next_github_release_version"
+changelog_link="**Full Changelog**: https://github.com/Fbada006/ArtMaker/compare/$previous_github_release_version...$new_github_release_version"
+
+# Add the Release Notes...
+full_github_release_notes="**New Feature**\n$refined_github_release_notes\n\n\n$changelog_link"
+
+# Get the confirmation before creating the GitHub Release...
+echo "The following GitHub Release will be created:"
+echo "GitHub Release Version: $next_github_release_version"
+echo "GitHub Release Notes: $github_release_notes"
+read -rp "Proceed? (y/n): " confirmation
+
+# Terminate the Bash Script's execution if either "n" or an invalid response has been input...
+if [[ $confirmation != "y" ]]; then
+    echo "GitHub Release Creation has been aborted..."
+    exit 0
+fi
+
+# Create the GitHub Release using the GitHub Rest API...
+github_repository="Fbada006/ArtMaker"
+github_api_response=$(curl -s -X POST "https://api.github.com/repos/$github_repository/releases" \
+                -H "Authorization: token $GITHUB_PERSONAL_ACCESS_TOKEN" \
+                -H "Content-Type: application/json" \
+                -d "{
+                    \"tag_name\": \"v$next_github_release_version\",
+                    \"target_commitish\": \"main\",
+                    \"name\": \"🎉 Release $next_github_release_version\",
+                    \"body\": \"$full_github_release_notes\",
+                    \"draft\": false,
```

Comment:
Set it to draft first, then someone can decide to edit before elevating to full release.
```suggestion
                    \"draft\": true,
```